indico/modules/events/logs/client/js/components/LogEntryModal.jsx

```diff
@@ -62,6 +64,18 @@ export default class LogEntryModal extends React.Component {
     nextEntry();
   }
 
+  relatedEntries() {
+    const {currentViewIndex, entries, relatedEntries} = this.props;
+    const entry = entries[currentViewIndex];
+    const urlArgs = [];
+    Object.entries(entry.meta).forEach(function(item) {
```

Comment:
arrow function please :)

Reply:
Actually, I think you can use `qs.stringify(...)` to build the query string. Something like this

```jsx
const query = qs.stringify({meta: entry.meta}, {allowDots: true});
```